lib/game.rb

```diff
@@ -0,0 +1,69 @@
+require_relative './game_state'
+require_relative './game_io/console_io'
+require_relative './messages'
+require_relative './game_rules'
+require_relative './validator'
+require_relative './board_presenter/board_presenter'
+require_relative './board_presenter/string_board'
+
+class Game
+    include Validator
+    include Messages
+    include GameRules
+
+    attr_accessor :game_state, :game_io, :status, :input, :board_presenter
+
+    def initialize(game_io: ConsoleIO.new, board_size:, player_2:, board_presenter: StringBoard.new())
```

Comment:
I notice there are some spots where you're using `()` for functions with no arguments, and other spots where you aren't, is that deliberate?

Reply:
Not deliberate! Forgot to go through and check for it -- fixed in new PR